ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -63,6 +68,11 @@ module.exports = Marionette.LayoutView.extend({
   updateCurrentQuery(currentQuerySettings) {
     const searchForm = new SearchForm(currentQuerySettings.get('template'))
     const sharedAttributes = searchForm.transformToQueryStructure()
+    if (currentQuerySettings.get('defaultResultFormId')) {
```

Comment:
you should be able to remove all the changes in this file, `user` is a single instance that is shared by all the components so we do not need to set it again. 
In this method `currentQuerySettings` and `user.getQuerySettings()` are referencing the same model